control-plane-operator/controllers/hostedcontrolplane/olm/olm-collect-profiles.go

```diff
@@ -21,6 +21,9 @@ var (
 func ReconcileCollectProfilesCronJob(cronJob *batchv1.CronJob, ownerRef config.OwnerRef, olmImage, namespace string) {
 	ownerRef.ApplyTo(cronJob)
 	cronJob.Spec = olmCollectProfilesCronJob.DeepCopy().Spec
+	cronJob.Spec.JobTemplate.ObjectMeta.Labels = map[string]string{
+		config.NeedManagementKASAccessLabel: "true",
```

Comment:
Actually I'm not sure the job will be matched by the pod selector

policy.Spec.PodSelector = metav1.LabelSelector{
		MatchExpressions: []metav1.LabelSelectorRequirement{
			{
				Key:      NeedManagementKASAccessLabel,
				Operator: "DoesNotExist",
				Values:   nil,
			},
			{
				Key:      "name",
				Operator: "NotIn",
				Values:   []string{"aws-ebs-csi-driver-operator"},
			},
		},